src/profiler/profilerthread.cpp

```diff
@@ -235,10 +235,12 @@ void ProfilerThread::saveData()
 	for (auto i = used_addresses.begin(); i != used_addresses.end(); ++i)
 	{
 		int proclinenum;
+		int line;
 		std::wstring procfile;
 		PROFILER_ADDR addr = i->first;
 
 		const std::wstring proc_name = sym_info->getProcForAddr(addr, procfile, proclinenum);

```

Comment:
`getProcForAddr` already calls `getLineForAddr` at the end, though there might be some naming confusion (function line number vs. address line number).